bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/discovery/VelbusThingDiscoveryService.java

```diff
@@ -108,8 +108,10 @@ public void onPacketReceived(byte[] packet) {
             } else {
                 logger.debug("Unknown command '{}' to address '{}'.", String.format("%02X", command),
                         String.format("%02X", address));
+                return false;
```

Comment:
Discovery service not supporting this particular packet type does not mean that the handler has been disposed does it? And it probably doesn’t mean that any classes deriving from this class shouldn’t attempt to handle the packet, right?

On the other hand this function is missing any code to handle this being disposed. Does this work correctly without said code, unlike the various kinds of things?

Reply:
Yes, it's corrected.